### PR TITLE
Add `asmgame.hackclub.com`

### PR DESCRIPTION
**Pull request by AverseABFun**
and have no idea if it works

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -423,6 +423,17 @@ asiet:
   - ttl: 1
     type: CNAME
     value: kkharidev.github.io.
+asmgame:
+  - ttl: 1
+    type: ALIAS
+    value: hackclub.github.io/asmgame.
```
Comment (Muirrum) [inline hackclub.com.yaml:429]: Hey! DNS records can't contain paths (in this case, `/asmgame`). You should make this a CNAME to hackclub.github.io (see docs [here](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain))

Additionally, this TTL is too low. Try a larger value, like 3600 (5 minutes).

File: hackclub.com.yaml [hunk 2]
```diff
@@ -423,6 +423,17 @@ asiet:
...
+  - ttl: 1
+    type: A
+    values:
+      - 185.199.108.153
+      - 185.199.109.153
+      - 185.199.110.153
+      - 185.199.111.153
```
Comment (Muirrum) [inline hackclub.com.yaml:436]: What's the purpose of this A record?

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 2 above)
Comment (reesericci) [reply hackclub.com.yaml:436]: image
rdns brings up github, so my best guess would be like a backup A if alias wasn't supported/didn't work?

File: hackclub.com.yaml (hunk 1 above)
Comment (alx-alexpark) [reply hackclub.com.yaml:429]: Isnt 300 5min?

File: hackclub.com.yaml (hunk 2 above)
Comment (AverseABFun) [reply hackclub.com.yaml:436]: Yep, I'm just following Github's docs although it might not be necessary.

File: hackclub.com.yaml (hunk 1 above)
Comment (AverseABFun) [reply hackclub.com.yaml]: fixed these issues

**Review by Muirrum — APPROVED** [review]
LGTM!

**Review by reesericci — APPROVED** [review]
LGTM!

**Pull request merged**